src/executor/aggregate_executor.cpp

```diff
@@ -169,37 +173,31 @@ bool AggregateExecutor::DExecute() {
       } else {
         tuple->SetAllNulls();
       }
-      auto location = output_table->InsertTuple(tuple.get());
-      PL_ASSERT(location.block != INVALID_OID);
-
-      auto &manager = catalog::Manager::GetInstance();
```

Comment:
The `tuple` object here is a local variable and gets destructed right after we leave the if clause. It seems that we just allocate it and set it as zero without inserting it into any tile or table. Is there something wrong? Or we just don't need the tuple for empty group by result?